<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EC_MeshmoonTeleport</class>
 <widget class="QWidget" name="EC_MeshmoonTeleport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#EC_MeshmoonTeleport {
	background-color: transparent;
}

QFrame#frameContent {
	background-color: white;
	border: 1px solid rgb(207, 207, 207);
	border-top: 0px;
	border-radius: 0px;
}

QFrame#frameButtons {
	background-color: rgb(240,240,240);
	border: 1px solid rgb(207, 207, 207);
	border-top: 0px;
	border-radius: 0px;
	border-bottom-right-radius: 3px;
	border-bottom-left-radius: 3px;
}

QLabel, QPushButton, QLineEdit {
	font-family: &quot;Arial&quot;;
	font-size: 13px;
}

QLabel#labelMessage {
	font-size: 16px;
	font-weight: bold;
	background-color: white;
	padding: 4px;
	padding-left: 10px;
	padding-bottom: 0px;
	color: rgb(59, 59, 59);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameContent">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="labelIcon">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="adminotechcomponents-resources.qrc">:/images/icon-teleport.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelMessage">
        <property name="text">
         <string>Do you want to launch teleport?</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameButtons">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxAskAgain">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Don't ask me again</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonYes">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Yes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonNo">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>No</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="adminotechcomponents-resources.qrc"/>
 </resources>
 <connections/>
</ui>
